features2d.go

```diff
@@ -350,13 +355,213 @@ func NewSimpleBlobDetector() SimpleBlobDetector {
 	return SimpleBlobDetector{p: unsafe.Pointer(C.SimpleBlobDetector_Create())}
 }
 
+// NewSimpleBlobDetectorWithParams returns a new SimpleBlobDetector with custom parameters
```

Comment:
Can you please add the link to the OpenCV docs, same as that used by `NewSimpleBlobDetector()`? Thank you.

Reply:
Added